### ISSUE TITLE
Add `add`/`rm` commands to edit plugins.toml

### ISSUE THREAD

**Issue by rossmacarthur** [rossmacarthur/sheldon#2]
Probably should use [`toml-edit`](https://crates.io/crates/toml-edit) for this.

**Comment by rossmacarthur** [comment]
I'm thinking the following sort of CLI would be nice

```
sheldon add <name> <options>
```

Examples:

```
sheldon add pure --github rossmacarthur/pure --revision "v1.9.0" 
```

Additionally I think something like this could be auto converted to a GitHub plugin

```
sheldon add pure --git https://github.com/rossmacarthur/pure --revision "v1.9.0"
```

**Comment by rossmacarthur** [comment]
You should be able to specify the templates to apply 

```
sheldon add pure --github rossmacarthur/pure --revision v1.0.5 --apply source PATH --use '{{ name }}.plugin.zsh'
```

Lets not forget remote plugins

```
sheldon add derp --remote https://derpinator.com
```

**Comment by rossmacarthur** [comment]
Implemented in 140d171. This will be released in version 0.5.0.
